src/applications/appeals/10182/tests/utils/helpers.unit.spec.js

```diff
@@ -1,6 +1,39 @@
 import { expect } from 'chai';
 
-import { wantsToUploadEvidence } from '../../utils/helpers';
+import {
+  canUploadEvidence,
+  needsHearingType,
+  wantsToUploadEvidence,
+  isDirectReview,
+} from '../../utils/helpers';
+
+describe('canUploadEvidence', () => {
+  it('should return false', () => {
+    expect(canUploadEvidence({ boardReviewOption: '' })).to.be.false;
+    expect(canUploadEvidence({ boardReviewOption: 'nope' })).to.be.false;
+    expect(canUploadEvidence({ boardReviewOption: 'direct_review' })).to.be
+      .false;
+    expect(canUploadEvidence({ boardReviewOption: 'hearing' })).to.be.false;
+  });
+  it('should return false', () => {
```

Comment:
Should this say `should return true` (and similarly for the other tests that have `.to.be.true`)?

Reply:
Oops, yes!